src/c_api/c_api_ndarray.cc

```diff
@@ -428,3 +436,41 @@ int MXCachedOpRegisterOpHook(NDArrayHandle handle,
   op->RegisterOpHook(clbk, monitor_all);
   API_END();
 }
+
+int MXNDArrayIsDeferredComputeEnabled(int *curr) {
+  API_BEGIN();
+  *curr = Imperative::Get()->is_deferred_compute();
+  API_END();
+}
+
+int MXNDArraySetDeferredComputeEnabled(int deferred_compute, int *prev) {
```

Comment:
why dont we call this `SetDeferredCompute` like the python API above `set_deferred_compute`

Reply:
For consistency with autograd APIs, lets call it `MXNDArraySetIsDeferredCompute`